index.html

```diff
@@ -10,14 +10,14 @@
 <body class="container">
     <div class="header_container">
         <div>
-            <input type="image" src="MyLogo.svg" alt="MyLogo">
+            <input type="image" src="/img/MyLogo.svg" alt="MyLogo">
         </div>
-        <a href="#"><img src="menu.svg" alt="Menu"></a>
+        <a href="#"><img src="/img/menu.svg" alt="Menu"></a>
     </div>
-    <img class="arc" src="arc.svg" width="1" height="15" alt="arc">
-    <img class="bar" src="bar.svg" width="1" height="15" alt="bar">
-    <img class="c" src="c.svg" width="1" height="15" alt="curve">
-    <img class="circle" src="circle.svg" width="1" height="15" alt="circle">
+    <img class="arc" src="/img/arc.svg" width="1" height="15" alt="arc">
+    <img class="bar" src="/img/bar.svg" width="1" height="15" alt="bar">
+    <img class="c" src="/img/c.svg" width="1" height="15" alt="curve">
+    <img class="circle" src="/img/circle.svg" width="1" height="15" alt="circle">
```

Comment:
Good job for changing the path. But it still does not work. For correct it do like this (`add . before /img`):
(do this for all your image)
```
<img class="arc" src="./img/arc.svg" width="1" height="15" alt="arc">
    <img class="bar" src="./img/bar.svg" width="1" height="15" alt="bar">
    <img class="c" src="./img/c.svg" width="1" height="15" alt="curve">
    <img class="circle" src="./img/circle.svg" width="1" height="15" alt="circle">

```